### PR TITLE
Disable login screen for dev

### PR DESCRIPTION
**Pull request by Graywing13**
Add this to ur client.env:
```
REACT_APP_DEV_DISABLE_LOGIN="true"
```

### CODE REVIEW

File: client/src/redux/reducers/users/usersSlice.js [hunk 1]
```diff
@@ -3,6 +3,23 @@ import { handleAsyncAction } from '../../handleAsync'
...
 
+const DEV_DISABLE_LOGIN = process.env.REACT_APP_DEV_DISABLE_LOGIN === 'true'
+
+const initialLoginState = DEV_DISABLE_LOGIN
+  ? {
+      user: {
+        id: '64a7310aee0a85231209105d',
+        username: 'mypassis123',
+        accessToken:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6Im15cGFzc2lzMTIzIiwiaWF0IjoxNjg5MTM3NDgxfQ.1H23KSGA-K-5h1zQaPFc-HHcCgwY8ONqJMT87y3qrF4',
```
Comment (NLmeng) [inline client/src/redux/reducers/users/usersSlice.js]: use .env for this?

**Comment by NLmeng** [comment]
also, maybe we should create like a main account that is used for demo and stuff and use that token instead of this one

**Review by NLmeng — APPROVED** [review]

**Review by L0Lmaker — COMMENTED** [review]
What if we want to log into different accounts to use different DB states?

**Comment by L0Lmaker** [comment]
> What if we want to log into different accounts to use different DB states?

I guess in this case we would disable it in env?

**Review by vee-16 — APPROVED** [review]

**Review by L0Lmaker — APPROVED** [review]

File: client/src/redux/reducers/users/usersSlice.js (hunk 1 above)
Comment (Graywing13) [reply client/src/redux/reducers/users/usersSlice.js]: good idea ty, done

**Pull request merged**